microsoft-edge/devtools-guide-chromium/index.md

```diff
@@ -75,25 +75,25 @@ In **Dock side**, choose any of the following layout options.
     
 :::row:::  
    :::column span="":::  
-      :::image type="complex" source="./media/devtools-intro-docking-left.msft.png" alt-text="Choose Dock To Left" lightbox="./media/devtools-intro-docking-left.msft.png":::  
-         Choose **Dock To Left**  
+      :::image type="complex" source="./media/devtools-intro-docking-left.msft.png" alt-text="Select Dock To Left" lightbox="./media/devtools-intro-docking-left.msft.png":::  
```

Comment:
Global in this file and possibly others: lowercase UI commands to match latest UI; "Dock to left" at least at:  DevTools > ... > hover over the docking icons.   Also in Command Menu.
That's sort of related to this PR's purpose of updating screenshots to reflect latest UI.